manageiq-operator/pkg/helpers/cr_migration/20210503163000_initial_migration.go

```diff
@@ -0,0 +1,114 @@
+package cr_migration
+
+import (
+	miqv1alpha1 "github.com/ManageIQ/manageiq-pods/manageiq-operator/pkg/apis/manageiq/v1alpha1"
```

Comment:
why is our code versioned/labeled/named `v1alpha`?

Reply:
We haven't moved to v1 yet

Reply:
we have been shipping the operator for about a year now.  What other items need to get done in order for us to get the operator out of alpha mode?  /cc @Fryguy

Reply:
I think we just need to make it a priority and test the upgrade